inc/zoo/swar/SWAR.h

```diff
@@ -68,7 +78,21 @@ struct SWAR {
                 ~(~T(0) << NBits),
         // Use LowerBits in favor of ~MostSignificantBit to not pollute
         // "don't care" bits when non-power-of-two bit lane sizes are supported
-        LowerBits = MostSignificantBit - LeastSignificantBit;
+        LowerBits = MostSignificantBit - LeastSignificantBit,
+        MaxUnsignedLaneValue = LeastSignificantLaneMask;
+
+    template <typename U>
+    constexpr auto loadIntoLanes(const U (&values)[Lanes]) const noexcept {
```

Comment:
Please rename to `from_array`.  As the symmetrical operation to `to_array`.
Also, this is a factory, it ought to be `static`, a class-function, not instance.